params/hooks_libevm.go

```diff
@@ -82,6 +83,12 @@ func (r RulesExtra) ActivePrecompiles(existing []common.Address) []common.Addres
 
 	var addresses []common.Address
 	addresses = slices.AppendSeq(addresses, maps.Keys(precompiles))
+
+	// Add p256verify address if Granite is enabled
+	if r.AvalancheRules.IsGranite {
+		addresses = append(addresses, common.HexToAddress("0x0000000000000000000000000000000000000100"))
+	}
```

Comment:
can we keep the same pattern as above where we have each specific contracts for upgrades?

Reply:
btw we should not redefine the address here but instead use it from libevm's `PrecompiledContractsP256Verify` var.